ajax.php

```diff
@@ -53,5 +53,8 @@
 }
 
 header('Content-type: application/json');
-echo Tools::json_encode($result);
+if (version_compare(_PS_VERSION_, '1.5', '<'))
+	echo json_encode($result);
+else
+	echo Tools::json_encode($result);
```

Comment:
Actually, there is an error here. The real function to call is ```Tools::jsonEncode(...)```, which is available from PrestaShop 1.4.0.2.